### DISCUSSION TITLE
BUG: image captions are not preserved when downloading image

### DISCUSSION THREAD

**Discussion by reytracen** [immich-app/immich#14190] [Q&A]
Hello,
I could not find anything about that, 
 immich v1.118.2 (linux ubuntu docker),
When viewing an image and displaying information on the right pane, I enter an image description and if I download the image locally, the description are gones.

Am I doing something wrong?

thank you for your help

**Comment by bo0tzz** [comment]
Where would you expect to find the description?

**Reply by reytracen** [reply]
I'll be more precise, sorry.
When editing the description in immich I'd expected it to be in the files metadata's (XMP ot IPTC caption, I don't know which is best)
This way, the file is "autonomous" the description information is embedded in the file itself.

**Reply by dclobato** [reply]
Considering that Immich does not change the file once it is imported, I believe that to have the description embed in the image (IPTC or XMP) Immich should: 1) apply these metadata before download using exiftool or some other tool, or; 2) offer the option to download the sidecar with the image, so the user can apply the metadata herself.

**Reply by reytracen** [reply]
Thank you for your answer, 
To me, option 1 is awesome, because this suits all the different users tech level.
Option 2 is ok but it requires user to be experienced. If I download many filles, operation has to be done to the whole group of downloaded files.

**Reply by dclobato** [reply]
I believe that the ideal scenario is to have a configuration option on the settings page, so the user can choose between options 1 and 2 on the fly...

People that use other tools (Lightroom, e.g.) may benefit from a sidecar, while users that are downloading to send the asset to someone else will be better served by option 1.

I know that some embedding metadata on video is quite challenging, specially date/time because of lack of standardization on this metadata...

**Comment by reytracen** [comment]
In the file itself using exiftool

ni©rey

-------- Message d'origine --------
